aspect/intellij_info_impl.bzl

```diff
@@ -150,13 +150,19 @@ def get_source_jars(output):
         return [output.source_jar]
     return []
 
-def library_artifact(java_output):
+def library_artifact(java_output, rule_kind = None):
     """Creates a LibraryArtifact representing a given java_output."""
     if java_output == None or java_output.class_jar == None:
         return None
     src_jars = get_source_jars(java_output)
+
+    if rule_kind != None and rule_kind.startswith("scala"):
```

Comment:
This won't disable ijars in Bazel build, but Intellij will get full jars. Impact for Intellij project is that it will have to index larger jars. On the other hand, Intellij project without source navigation in large deployments is mostly useless.

Reply:
I suppose my question is how this will affect IJ performance overall, including project import. I'm not sure you'd always want all the depedencies' sources, and you might want to just navigate the project code itself. But absent a smarter mechanism, if this works for you it's good enough.